src/main/java/core/basesyntax/service/impl/DataConverterServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverterService;
+import java.util.List;
+
+public class DataConverterServiceImpl implements DataConverterService {
+    private static final int TITTLE_POSITION = 1;
+    private static final int NECESSARY_COUNT_OF_ELEMENTS = 3;
+    private static final String COMMA_REGEX = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        return inputReport.stream()
+                .skip(TITTLE_POSITION) //for skipping title
```

Comment:
The constant `TITTLE_POSITION` is being used to skip the first line assuming it is a title, but it seems to be misnamed as it is set to 1 which is not an index but a count. If your intention is to skip the first line, the constant should be named appropriately and set to 0, because `stream().skip(n)` skips the first `n` elements.